.changeset/many-tables-love.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Curious: does this deserve its own changeset? 

I know excluding them ended up being part of the perf gains but not sure if we should have a separate entry for that, but i may be wrong

wdyt @MarcosSpessatto ?

Reply:
We have a customer asking for this specific change, that's why I decided to include this here
That's described in SUP-558